met-web/src/components/layout/Footer/index.tsx

```diff
@@ -1,165 +1,231 @@
+                        <BCLogo
+                            style={{
+                                alignSelf: 'flex-start',
+                                paddingBottom: '1rem',
+                                maxWidth: isMediumScreenOrLarger ? '25rem' : '9.375rem',
                             }}
                             alt={translate('footer.defaultLogo')}
                         />
-                    </Grid>
-                    <Grid item xs={12}>
-                        <MetBodyOld>{translate('footer.connectWithUs')}</MetBodyOld>
-                    </Grid>
+
+                        {/* Footer accessibility paragraph and link */}
+                        <p style={{ margin: '0' }}>{translate('footer.statement')}</p>
+                        <Stack
+                            direction="row"
+                            alignSelf="flex-end"
+                            style={{ height: isMediumScreenOrLarger ? '100%' : 'auto', width: '100%' }}
+                        >
+                            <Link
+                                href="https://www2.gov.bc.ca/gov/content/home/accessible-government"
+                                sx={{
+                                    display: 'flex',
+                                    alignItems: 'center',
+                                    alignSelf: 'flex-end',
+                                    ...linkStyles,
+                                }}
+                            >
+                                <FontAwesomeIcon
+                                    style={{
+                                        fontSize: '2.5rem',
+                                        color: Palette.text.primary,
+                                        paddingRight: '0.75rem',
+                                        width: '2.5rem',
+                                    }}
+                                    icon={faUniversalAccess}
+                                />
+
+                                {translate('footer.accessibility')}
+                                <LinkArrow />
+                            </Link>
+                        </Stack>
+                    </Stack>
                 </Grid>
+
+                {/* Footer more info */}
                 <Grid
                     item
-                    xs={12}
-                    md={6}
+                    columns={10}
+                    xs={10}
+                    md={2}
                     container
-                    justifyContent={'flex-start'}
-                    alignItems="flex-start"
-                    rowSpacing={3}
+                    sx={{
+                        height: isMediumScreenOrLarger ? '15.625rem' : 'auto',
+                        pr: isMediumScreenOrLarger ? '1rem' : '0',
+                    }}
                 >
-                    <Grid item xs={12}>
+                    <Stack spacing={3}>
                         <MetLabel>{translate('footer.moreInfo')}</MetLabel>
```

Comment:
I'd recommend using a span or even a heading tag here instead. Labels are usually just used with inputs and form fields.